src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -2919,6 +2914,23 @@ private BoundExpression BindDelegateCreationExpression(ObjectCreationExpressionS
                     return new BoundDelegateCreationExpression(node, boundLambda, methodOpt: null, isExtensionMethod: false, type: type, hasErrors: !conversion.IsImplicit);
                 }
 
+                else if (analyzedArguments.HasErrors)
```

Comment:
Because I want to limit my change for only unbound lambda. Only make unbound lambda better. Other code, no better, no worse, same as before.